analyses/WA_cd_2010/doc_WA_cd_2010.md

```diff
@@ -0,0 +1,23 @@
+# 2010 Washington Congressional Districts
+
+## Redistricting requirements
+In Washington, districts must:
+
+1. be contiguous
+1. have equal populations
+1. be geographically compact
+1. preserve county and municipality boundaries as much as possible
+
+
+### Interpretation of requirements
+We enforce a maximum population deviation of X.X%.
+
+## Data Sources
+Data for Washington comes from the ALARM Project's [2020 Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans for Washington.
+No special techniques were needed to produce the sample.
```

Comment:
You need to fill in the details of your simulation approach here.  See the WA 2020 example.